massa-execution-exports/Cargo.toml

```diff
@@ -27,7 +27,7 @@ massa_final_state = { path = "../massa-final-state" }
 massa_ledger_exports = { path = "../massa-ledger-exports", optional = true }
 massa_module_cache = { path = "../massa-module-cache" }
 massa_versioning = { path = "../massa-versioning" }
-massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", branch = "main" }
+massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", branch = "feature/Improve_ABI_types_in_wasmv1" }
```

Comment:
/!\ Do not merge until we:
- Merge the massa-sc-runtime common branch into main
- Update the Cargo.toml with a rev of the massa-sc-runtime commit